bodhi/server/tasks/tag_update_builds.py

```diff
@@ -36,16 +36,19 @@ def main(update: Update, builds: typing.List[str]):
     """
     koji = buildsys.get_session()
     koji.multicall = True
-    for build in builds:
-        if update.from_tag:
-            # this is a sidetag based update. use the sidetag pending signing tag
-            side_tag_pending_signing = update.release.get_pending_signing_side_tag(update.from_tag)
-            koji.tagBuild(side_tag_pending_signing, build)
-        elif update.release.pending_signing_tag:
-            # Add the release's pending_signing_tag to all new builds
-            koji.tagBuild(update.release.pending_signing_tag, build)
-        else:
-            # EL6 doesn't have these, and that's okay...
-            # We still warn in case the config gets messed up.
-            log.warning(f'{update.release.name} has no pending_signing_tag')
-    koji.multiCall()
+    db_factory = util.transactional_session_maker()
+    with db_factory() as session:
+        update = session.query(Update).filter_by(alias=update["alias"]).first()
```

Comment:
Should we care for when update is ``None`` (ie: the alias provided did not link to an entry in the DB)?
If the worker are over-loaded, we could have a race condition between when the task was put in the queue and when the worker gets to it (and have someone delete the update in between).

Reply:
I don't think it is possible to delete an update other than directly from the database. I am not sure it is really needed to handle that case ? What do you think ?

Reply:
I thought it was, maybe it did and got removed.
That does lower the chances of running into this quite a bit indeed.

Let me know if you want me to approve the PR as is or if you want to update it